core/src/main/res/values/strings.xml

```diff
@@ -790,56 +790,56 @@
+    <string name="notification_channel_downloading">Downloading…</string>
+    <string name="notification_channel_downloading_description">Displayed while downloading.</string>
     <string name="notification_channel_playing">Currently playing</string>
-    <string name="notification_channel_playing_description">Allows to control playback. This is the main notification you see while playing a podcast.</string>
+    <string name="notification_channel_playing_description">Allows playback control. The main notification displayed while playing a podcast.</string>
     <string name="notification_channel_download_error">Download failed</string>
-    <string name="notification_channel_download_error_description">Shown when download or feed update fails.</string>
-    <string name="notification_channel_sync_error">Synchronization failed</string>
-    <string name="notification_channel_sync_error_description">Shown when gpodder synchronization fails.</string>
-    <string name="notification_channel_auto_download">Automatic download completed</string>
-    <string name="notification_channel_episode_auto_download">Shown when episodes have been automatically downloaded.</string>
+    <string name="notification_channel_download_error_description">Displayed when a download or feed update fails.</string>
+    <string name="notification_channel_sync_error">Syncing failed</string>
+    <string name="notification_channel_sync_error_description">Displayed when gpodder.net syncing fails.</string>
+    <string name="notification_channel_auto_download">Auto-download completed</string>
+    <string name="notification_channel_episode_auto_download">Displayed when episodes are auto-downloaded.</string>
     <string name="notification_channel_new_episode">New Episode</string>
-    <string name="notification_channel_new_episode_description">Shown when a new episode of a podcast was found, where notifications are enabled</string>
+    <string name="notification_channel_new_episode_description">Displayed when for new podcast episodes</string>
 
     <!-- Widget settings -->
-    <string name="widget_settings">Widget settings</string>
-    <string name="widget_create_button">Create widget</string>
+    <string name="widget_settings">Widgets</string>
+    <string name="widget_create_button">New widget</string>
     <string name="widget_opacity">Opacity</string>
 
     <!-- On-Demand configuration -->
-    <string name="on_demand_config_setting_changed">Setting updated successfully.</string>
-    <string name="on_demand_config_stream_text">Looks like you stream a lot. Do you want episode lists to show stream buttons?</string>
-    <string name="on_demand_config_download_text">Looks like you download a lot. Do you want episode lists to show download buttons?</string>
+    <string name="on_demand_config_setting_changed">Settings updated.</string>
+    <string name="on_demand_config_stream_text">Do you want stream buttons available in episode lists?</string>
```

Comment:
I think it is good to say how AntennaPod came to the conclusion that it needs to ask that question. Please revert

Reply:
It is creepy.